satinfo/amsua_n19/readme

```diff
@@ -1,3 +1,6 @@
-n19  20091215 000000  20091221 240000  amsua 14  1 2 3 4 5 6 7 8 9 10 11 12 13 14 15               # start date per Merra 
-n19  20091222 000000  20150114 060000  amsua 13  1 2 3 4 5 6 7   9 10 11 12 13 14 15               # channel 8 noisy per Merra
-n19  20150114 120000  21001231 240000  amsua 12  1 2 3 4 5 6     9 10 11 12 13 14 15               # turn off channel 7 as in operational satinfo
+# launched 20090209, dumps started 20009293, GSI started to use 20091215
```

Comment:
typo. "20009293" is not a valid date and i cant figure out what date you meant.

Reply:
should be 20090929 (fixed).  Info comes from historical/Satellite_Historical_Documentation.txt